tools/toolchain/scripts/install_sirius.sh

```diff
@@ -50,113 +54,113 @@ case "$with_sirius" in
+                             https://www.cp2k.org/static/downloads/SIRIUS-${sirius_ver}.tar.gz
             fi
             echo "Installing from scratch into ${pkg_install_dir}"
             [ -d sirius-${sirius_ver} ] && rm -rf sirius-${sirius_ver}
             tar -xzf SIRIUS-${sirius_ver}.tar.gz
             cd SIRIUS-${sirius_ver}
-            cat <<EOF > make.inc
-CXX = \$(MPICXX)
-BASIC_CXX_OPT = -O3 -DNDEBUG -mtune=native -ftree-loop-vectorize ${MATH_CFLAGS}
-CXX_OPT = \$(BASIC_CXX_OPT) -fopenmp -std=c++11 -D__SCALAPACK -D__ELPA
-CXX_OPT := \$(CXX_OPT) -I${PWD}/src
-CXX_OPT := \$(CXX_OPT) -I${PWD}/src/SDDK
-CXX_OPT := \$(CXX_OPT) ${ELPA_CFLAGS}
-CXX_OPT := \$(CXX_OPT) ${GSL_CFLAGS}
-CXX_OPT := \$(CXX_OPT) ${SPGLIB_CFLAGS}
-CXX_OPT := \$(CXX_OPT) ${HDF5_CFLAGS}
-MPI_FC = ${MPIFC}
-MPI_FC_OPT = -g -O2 -fopenmp -cpp
-
-LIBS := ${LIBXC_LDFLAGS} ${FFTW_LDFLAGS} ${ELPA_LDFLAGS} ${SCALAPACK_LDFLAGS} ${GSL_LDFLAGS} ${SPG_LDFLAGS}
-LIBS := \$(LIBS) ${SPGLIB_LIBS} ${FFTW_LIBS} ${GSL_LIBS} ${SCALAPACK_LIBS} ${ELPA_LIBS}
-EOF
+            cd src && sed -i -e "s/desciption/description/g" options.json && cd ..
+            rm -Rf build
+            mkdir build
+            cd build
+            COMPILATION_OPTIONS="-DHDF5_DIR=${HDF5_DIR}"
+            if [ -n "$ELPA_LIBS" ] ; then
+                if [ -s "$ELPAROOT" ] ; then
+                    export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$ELPAROOT/lib/pkgconfig:$ELPAROOT/lib64/pkgconfig
+                fi
+                COMPILATION_OPTIONS="-DUSE_ELPA=ON -DELPA_INCLUDE_DIR=${ELPAROOT}/include/elpa-${ELPAVERSION}/elpa $COMPILATION_OPTIONS"
+            fi
 
-            # this a hack needs to be fixed permanently
-            sed -i -e "s/: log/: /g" src/Makefile
-            cat > src/version.hpp <<EOF
-#ifndef __VERSION_H__
-#define __VERSION_H__
-const char* const git_hash = "6ec392682d7ffda6090ee3124d6841adc6a5fd0d";
-const char* const git_branchname = "heads/develop";
-const char* const build_date = "Thu, 23 Aug 2018 13:58:38";
-#endif
-EOF
+            echo "scalapack : $SCALAPACK_CFLAGS"
+            if [ -n "$SCALAPACK_LIBS" ] ; then
+                export SCALAPACK_LIB="$SCALAPACK_LIBS"
+                if [ -s "$SCALAPACKROOT" ] ; then
+                    COMPILATION_OPTIONS="-DUSE_SCALAPACK=ON -DSCALAPACK_INCLUDE_DIR=${SCALAPACKROOT}/include ${COMPILATION_OPTIONS}"
+                else
+                    COMPILATION_OPTIONS="-DUSE_SCALAPACK=ON ${COMPILATION_OPTIONS}"
+                fi
+            fi
+            if [ -n "$HDF5_LIBS" ] ; then
+                COMPILATION_OPTIONS="-DUSE_HDF5=ON $COMPILATION_OPTIONS"
+            fi
+            if [ -n "$LIBVDWXC_LIBS" ] ; then
+                COMPILATION_OPTIONS="-DUSE_VDWXC=ON $COMPILATION_OPTIONS"
+            fi
+            if [ -n "$MKL_LIBS" ] ; then
+                COMPILATION_OPTIONS="-DUSE_MKL=ON -DUSE_SCALAPACK=ON $COMPILATION_OPTIONS"
+            fi
+
+            echo "cmake -DCMAKE_INSTALL_PREFIX=${pkg_install_dir} \
```

Comment:
Please remove.

Reply:
the echo is useful to know what is given to cmake I really do not understand why you do not want this information.

Reply:
Well, actually I don't care too much. It's just that so fare we tried to keep the toolchain rather quiet and it looked like a leftover debug statement. I'm ok if you deliberately want to leave it in.

Reply:
I just removed them. It is practical to have them though at least the command because I spend quite some time to understand why it was not working out of the box the first time inside the CI while working on my machine.

The if statements are mandatory though

Reply:
Thanks.